vpn-router-bootstrap/src/main/resources/application.yml

```diff
@@ -1,2 +1,4 @@
 storage:
   clients-file-path: ${HOME}/vpn-router/storage/clients.yml
+network:
+  address: 192.168.0.0/24
```

Comment:
You know my knowledge of computer networks is limited) I googled what 192.168.0.0/24 means. And as I understand, 192.168.0.0 is a standard ip address for many routers and /24 means that we expect ranges between 192.168.0.0 and 192.168.0.255. Am I right?

Reply:
🔥